src/components/header.css

```diff
@@ -0,0 +1,63 @@
+.header {
+  width: 100%;
+  height: 100px;
+  position: fixed;
+}
+
+.headerContainer {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 10rem;
+  height: 100px;
+  align-items: center;
+}
+
+.navList {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.linkLi {
+  list-style: none;
+  margin: 0;
+  margin-left: 1rem;
```

Comment:
Nothing urgent here, but as we start to see some patterns in spacing (if there are any), let's set those up as variables as well.